libs/templates/src/lib/trade/index.tsx

```diff
@@ -11,9 +11,8 @@ export function TradeTemplate() {
           title="Leveraged forex on Deriv MT5"
           description={`Deriv MT5 offers various trading assets on the forex market via a single platform.`}
```

Comment:
```suggestion
          description={'Deriv MT5 offers various trading assets on the forex market via a single platform.'}
```

Can you update this as well?